### PR TITLE
add OWNERs file

### PR DESCRIPTION
**Pull request by kim-tsao**
### What does this PR do?:

Adds an OWNERs file

### Which issue(s)/story(ies) does this PR fixes:

https://issues.redhat.com/browse/DEVHAS-386

### PR acceptance criteria:

- [ ] Unit/Functional tests

  

- [ ] Documentation 

   

- [ ] Client Impact

### CODE REVIEW

**Review by keithchong — APPROVED** [review]
LGTM

**Review by johnmcollier — APPROVED** [review]
Looks good. But @keithchong should approve too before merging.

**Pull request merged**
